### DISCUSSION TITLE
[ZK Stack] - with newer versions, Zk Stack Init does not init the .toml file

### DISCUSSION THREAD

**Discussion by TicTacWeb3** [zkSync-Community-Hub/zksync-developers#497] [ZK Stack] [answered]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [ ] zkEVM (era-zk_evm)
- [X] Server (zksync-era)
- [ ] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [ ] Block Explorer (clock-explorer)
- [ ] Portal - Wallet + Bridge (dapp-portal)
- [ ] Other (please specify below)

### Rust Version

installed toolchains :
 stable-x86_64-unknown-linux-gnu (default) nightly-2023-08-21-x86_64-unknown-linux-gnu nightly-2024-02-01-x86_64-unknown-linux-gnu  active toolchain
 stable-x86_64-unknown-linux-gnu (default) rustc 1.77.2 (25ef9e3d8 2024-04-09)

### Operating System

- [ ] Ubuntu
- [ ] MacOS
- [ ] Windows
- [X] Other (Please specify below)

### Issue Description

on EndeavourOS (an Arch-based distro) and NixOs, when i start with zk stack init and i cofigure a new chain, right after i finished configuring, nothing start and it just gives me the error "yourchainname.toml was not found"

### Expected Behavior

the expected behaviour is that when i configure my chain, a toml file needs to be generated, this worked with previous versions

### Current Behavior

the current behaviour is this :
>[endre@Syntra zksync-era]$ zk stack init
 > Initializing hyperchain creation
✔ Do you want to configure a new chain or use an existing configuration? · Configure new chain
> Setting hyperchain configuration
✔ What is your hyperchain name? · ExampleChain
✔ What is your hyperchain id? Make sure this is not used by other chains. · 23432343212343
✔ To which L1 Network will your hyperchain rollup to? · localhost (matterlabs/geth)
Configs compiled for dev
Configs compiled for dev
WARN[0000] The "GITHUB_WORKSPACE" variable is not set. Defaulting to a blank string. 
WARN[0000] /home/endre/Scrivania/SyntraNet/zksync-era/docker-compose.yml: `version` is obsolete 
[+] Running 2/0
 ✔ Container zksync-era-postgres-1  Running                            0.0s 
 ✔ Container zksync-era-reth-1      R...                               0.0s 
> Ensuring databases are up
✔ Ensuring databases are up done (13ms)
Deployer
Address - 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
Governor
Address - 0xa61464658AfeAf65CccaaFD3a512b69A83B77618
ETH Operator
Address - 0x0D43eB5B8a47bA8900d84AA36656c92024e9772e
Blob Operator
Address - 0xA13c10C0D5bd6f79041B9835c63f91de35A15883
Fee receiver
Address - 0x8002cD98Cfb563492A6fB3E7C8243b7B9Ad4cc92
Private keys for these wallets are available in the .env file for you chain. Make sure that you have a copy in a safe place.

and right after it gives me this error...

**_File configs/examplechain.toml not found_**

[endre@Syntra zksync-era]$ 

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by bxpana** [comment]
Thanks for reaching out, we'll take a moment to review and get back to you

**Comment by zk-Lumi** [comment, marked as answer]
Thanks for reaching out @TicTacWeb3. A new and improved ZK Stack CLI is coming very soon (internally tested currently, should be live end of month / start of June), which will fix this issue. The current version is [deprecated](https://docs.zksync.io/zk-stack/running-a-hyperchain/locally.html) and may not function as expected.

Apologies for the inconvenience, the new CLI will be a production-ready stable release.
